packages/typeorm/src/services/connections.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/typeorm
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  BindingScope,
+  ContextTags,
+  CoreBindings,
+  inject,
+  LifeCycleObserver,
+  lifeCycleObserver,
+} from '@loopback/core';
+import {ConnectionManager} from 'typeorm';
+import {ApplicationUsingTypeOrm, TypeOrmBindings} from '../';
+
+@lifeCycleObserver('datasource', {
+  tags: {[ContextTags.KEY]: TypeOrmBindings.CONNECTIONS},
+  scope: BindingScope.SINGLETON,
+})
+export class TypeOrmConnections implements LifeCycleObserver {
```

Comment:
As mentioned elsewhere, a class should have a singular name. At minimum, use something like `TypeOrmConnectionManager`. Preferably, get rid of this class entirely as discussed in https://github.com/strongloop/loopback-next/pull/5801/files#r444168243